src/components/table/template.njk

```diff
@@ -0,0 +1,27 @@
+<table class="govuk-c-table {{ classes }}">
+  {% if caption %}
+  <caption class="govuk-c-table__caption {{captionSize}}">{{ caption }}</caption>
```

Comment:
Are there issues if `captionSize` is not set?

Also, if caption size is `heading-small` as in the example docs, the class added will just be `heading-small`. Is that a valid class, or should it be a modifier? Either way, would it make sense to allow the user to set just `'captionSize': 'small'` and add the rest of the class name here?